lib/cmds/start.ts

```diff
@@ -23,8 +24,10 @@ let prog = new Program()
     .addOption(Opts[Opt.VERSIONS_STANDALONE])
     .addOption(Opts[Opt.VERSIONS_CHROME])
     .addOption(Opts[Opt.VERSIONS_ANDROID])
+    .addOption(Opts[Opt.VERSIONS_APPIUM])
     .addOption(Opts[Opt.CHROME_LOGS])
     .addOption(Opts[Opt.ANDROID])
+    .addOption(Opts[Opt.IOS])
```

Comment:
Check to see if the user is on os.type() === 'Darwin' before adding the iOS options.